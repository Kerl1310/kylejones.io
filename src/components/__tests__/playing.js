import React from 'react';
import renderer from 'react-test-renderer';

import Playing from '../now/playing';

const testGame = {
    steamId: '220',
    name: 'Half Life 2'
}

describe('Playing', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<Playing game={testGame}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
