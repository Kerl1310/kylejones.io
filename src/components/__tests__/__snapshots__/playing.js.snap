// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Playing renders correctly 1`] = `
<li>
  <strong>
    Playing
    : 
  </strong>
  <a
    className="game-link"
    href="https://store.steampowered.com/app/220"
    rel="noopener noreferrer"
    target="_blank"
  >
    Half Life 2
  </a>
</li>
`;
